### DISCUSSION TITLE
Backup and restore earnapp data

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#124] [Q&A]
Hello,
As far as I know, when I move to new server, I just need to backup earnapp.txt & proxies.txt. In new server, I just need to restore these file and start it. Is it right? Thanks

**Comment by engageub** [comment]
Hello,
Yes, You just need to copy those files for earnapp to restore and start. 
The file earnapp.txt is not deleted by using --delete. 
To delete the backup files, you have to use --deleteBackup option. 

Thank you
